driver/pparse.mli

```diff
@@ -28,6 +28,19 @@ type 'a ast_kind =
 | Structure : Parsetree.structure ast_kind
 | Signature : Parsetree.signature ast_kind
 
+val clear_ppx : unit -> unit
```

Comment:
Absence of this line slightly changes semantics. Before my changes `Cflag.all_ppx` was used to store all information about external ppxs and code like `all_ppx :=...` was supposed to remove old information and replace it by new one. Now information about ppx extensions is also stored in `Pparse module` and `Cflags.all_ppx` is kind of .... redundant? Can we get rid of it? What about backward compatibility?

Also if I remove `clear_ppx` call something bad can happen if `readenv` will be called twice. (I think)